timeseries/template.timeseries

```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env python
-{{ batchdirectives }}
+#PBS -N timeseries.{{ case }}
+#PBS  -r n 
+#PBS  -j oe 
+#PBS  -V 
+#PBS -l select=10:ncpus=40:mpiprocs=10
```

Comment:
@jedwards4b - This should stay as a template variable {{ batchdirectives }} for
portability. Is there are reason that you made it specific to PBS?